images/filebeat/tests/test.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+container_name="filebeat-${RANDOM}"
+
+docker run -d \
+  --name=${container_name} \
+  --user=nonroot \
+  --volume="$(pwd)/filebeat.yml:/usr/share/filebeat/filebeat.yml:ro" \
+  --volume="/var/lib/docker/containers:/var/lib/docker/containers:ro" \
+  --volume="/var/run/docker.sock:/var/run/docker.sock:ro" \
```

Comment:
I don't love this, is this so filebeat can introspect the logs on the other containers running on the host? does it actually need the socket for this or is just the `/var/lib/` enough?